core/src/lib/parse_conf.cc

```diff
@@ -507,6 +503,13 @@ ConfigurationParser::BackupResourceTable()
   return backup_table;
 }
 
+std::shared_ptr<ConfigResourcesContainer>
+ConfigurationParser::GetResourcesTablePointer()
```

Comment:
It is `BackupResourceTable()` and `RestoreResourceTable()` so it should be singular here, too.
The function does not actually return a pointer, so `...Pointer()` feels misleading. Maybe `...Handle()` is better or maybe drop the suffix completely and just call it `GetResourceTable()`.

Reply:
Makes sense, done.